sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/data/PointData.java

```diff
@@ -38,4 +39,8 @@
    * @return the attributes associated with this {@code Point}.
    */
   Attributes getAttributes();
+  /**
+   * (Optional) List of exemplars collected from measurements that were used to form the data point.
```

Comment:
```suggestion
   * List of exemplars collected from measurements that were used to form the data point.
```

Optional conflates with the ability to be null. We generally don't call out the fact that this could be empty but you can add wording using `empty` if you want

Reply:
The comment is directly copy-pasted from the spec/proto.  I can remove it if that's the convention in Java.

Reply:
Yeah - optional in the spec generally refers to whether a method exists at all. Given the method is here, now the SDK has implemented an optional part of the spec. Spec should generally not be copied into the SDK since it's target audience is SDK authors, not end users.